test/recipes/80-test_ssl_old.t

```diff
@@ -475,8 +475,13 @@ sub testssl {
                 $flag = "-tls1_2";
             }
             foreach my $cipher (@{$ciphersuites{$protocol}}) {
-                ok(run(test([@ssltest, @exkeys, "-cipher", $cipher, $flag])),
-                "Testing $cipher");
+                if ($protocol eq "SSLv3" && $cipher =~ /EC?DH/ ) {
```

Comment:
We probably should not disable any EDH tests, just ECDH should probably suffice.  So I would drop the "?".